eagle-server/src/main/webapp/app/dev/public/js/ctrls/alertEditCtrl.js

```diff
@@ -304,6 +304,15 @@
 			return false;
 		};
 
+
+		$scope.checkPolicyName = function () {
+			if($scope.policy.name.length > 50) {
+				return "length should less than 50";
+			}
+			return false;
```

Comment:
Yes,I agree with you.But I see checkPublisherName return false. So I consistent with checkPublisherName  method.